cypress/e2e/po/pages/login-page.po.ts

```diff
@@ -44,7 +44,7 @@ export class LoginPagePo extends PagePo {
     });
   }
 
-  private submitButton(): AsyncButtonPo {
+  submitButton(): AsyncButtonPo {
```

Comment:
why was this needed?

Reply:
so that I can use it on the spec, otherwise it would be unreachable outside it's PO:
```
loginPage.submitButton().self().should('have.css', 'background').should((background: string) => { ... etc
```